wkhtmltopdf/utils.py

```diff
@@ -60,12 +61,16 @@ def _options_to_args(**options):
         formatted_flag = '--%s' % name if len(name) > 1 else '-%s' % name
         formatted_flag = formatted_flag.replace('_', '-')
         accepts_no_arguments = formatted_flag in NO_ARGUMENT_OPTIONS
+        is_multi_value_option = formatted_flag in MULTI_VALUE_OPTIONS
         if value is None or (value is False and accepts_no_arguments):
             continue
         flags.append(formatted_flag)
         if accepts_no_arguments:
             continue
-        flags.append(six.text_type(value))
+        if is_multi_value_option:
+            flags.extend(value.split())
```

Comment:
We might want to do `value.split(maxsplit=1)` to respect key/values of which the value contains a space, e.g. `Authorization Basic Ym9zY236Ym9zY28=`

HTTP Authorization headers often contain a space in the value part to identify the type of credential (Basic, Bearer, Digest, etc). Header keys never contain spaces, so `maxsplit=1` should be sufficient.

Reply:
Nice suggestion